.gitmodules

```diff
@@ -7,7 +7,7 @@
 	url = https://github.com/tommie/lx106-hal
 [submodule "esptool"]
 	path = esptool
-	url = https://github.com/pfalcon/esptool
+	url = https://github.com/themadinventor/esptool.git
```

Comment:
The "--version=2" changes are needed. Without them there is no way to create a >512KB image (like the one sgi tests produce). Of course we could create a linker script which would create canonical 0x00000/0x40000 image of 1MB+ size but using OTA images and 1/2/4MB linker scripts already provided by Espressif seems to be a better solution, doesn't it?